cli/module_registry.go

```diff
@@ -209,21 +200,26 @@ func UploadModuleAction(c *cli.Context) error {
 				"if you want to upload a version without a meta.json, you must supply a module name and namespace (or module name and org-id)",
 			)
 		}
-		moduleID, err = updateManifestModuleIDWithArgs(c, client, nameArg, publicNamespaceArg, orgIDArg)
-		if err != nil {
-			return err
-		}
+		// get from the BE?
 	} else {
 		// if we can find a manifest, use that
 		manifest, err := loadManifest(manifestPath)
 		if err != nil {
 			return err
 		}
 
-		moduleID, err = updateManifestModuleIDWithArgs(c, client, manifest.Name, publicNamespaceArg, orgIDArg)
-		if err != nil {
-			return err
+		if len(manifest.ModuleID) > 0 {
```

Comment:
```suggestion
		if manifest.ModuleID != "" {
```

[nit] if we are going to use len, we should at least be consistent